README.rst

```diff
@@ -30,9 +30,9 @@ To run the tests::
 
  $ python -m pytest
 
-To see a coverage report, check pep8 and pyflakes::
+To see a coverage report, check pytest-cov::
```

Comment:
I thought that we use flake8 instead of pep8 and pyflakes.
So perhaps:
```
To see a coverage report, check pytest-cov:
 $ python -m pytest --cov-report term-missing --cov=pytac

To see style violations, use flake8:
 $ flake8
```
Or we could use [this](https://github.com/tholo/pytest-flake8) plugin:
```
To see a coverage report and style violations, you can use pytest-cov and flake8:
 $ python -m pytest --cov-report term-missing --cov=pytac --flake8
```